pkg/providers/nutanix/config/cp-template.yaml

```diff
@@ -1,10 +1,21 @@
+apiVersion: v1
+binaryData:
+  ca.crt: {{.nutanixAdditionalTrustBundle}}
```

Comment:
i am not sure if thats true. per https://github.com/nutanix-cloud-native/cluster-api-provider-nutanix/blob/main/pkg/client/client.go#L157 and https://github.com/nutanix-cloud-native/prism-go-client/blob/main/v3/v3.go#L40 its going all the way in pem format. only for tests https://github.com/aws/eks-anywhere/blob/main/internal/pkg/api/nutanix.go#L107 its was encoded in base64 per https://github.com/nutanix-cloud-native/cluster-api-provider-nutanix/pull/171

Reply:
thanks @abhinavmpandey08 with the debugging. seems like being configmap with binarydata it expects the data in base64 https://kubernetes.io/docs/concepts/configuration/configmap/#configmap-object. will find a fix. also we replaced binaryData to data but it was failing due to multiline char missing and indentation required
<pre>ca.crt: |
     ---
     ---
<pre>

Reply:
another thing suggested by @abhinavmpandey08 is that we should move this configmap into a separate file as when eks-a calls delete on the yaml, configmap gets deleted first and deletion of vm fails. once we move it to another file like how secret-template.yaml is, delete on it is never called and hence it can be used during deletion